app/lib/getFileIcon/windows.js

```diff
@@ -0,0 +1,24 @@
+import extractIcon from 'win-icon-extractor'
+
+/**
+ * Get system icon for file
+ *
+ * @param  {String} path File path
+ * @param  {Number} options.width
+ * @param  {[type]} options.height
+ * @return {Promise<String>} Promise resolves base64-encoded source of icon
+ */
+export default function getFileIcon(path, { width = 128, height = 128 } = {}) {
```

Comment:
You are not using second argument (object with width/height), so you can remove it (+ update doc of this method)